spec/features/user_clicks_on_vote_spec.rb

```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 describe 'When a user clicks on Vote' do
-  it 'they are taken to the project index' do
+  xit 'they are taken to the project index' do
+    # i think this is gone because of the new redirect?
```

Comment:
Makes sense. Let's delete it. Is there another one to check clicking on 'Submit new project'? I think I was stumbling around and figured writing small tests would help me get started.